### PR TITLE
picotool 1.1.2

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

- [x] `resource` blocks have been checked for updates.
<details>
  <summary>release notes</summary>
  <pre>Small release with one bug fix to allow specifying `0` for `--bus`, and some minor documentation edits.

Thanks to the following for their contributions:

Andrew Scheller
XDeschuyteneer
</pre>
</details>

### CODE REVIEW

File: Formula/picotool.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Picotool < Formula
   license "BSD-3-Clause"
 
   stable do
-    url "https://github.com/raspberrypi/picotool/archive/refs/tags/1.1.1.tar.gz"
-    sha256 "2d824dbe48969ab9ae4c5311b15bca3449f5758c43602575c2dc3af13fcba195"
+    url "https://github.com/raspberrypi/picotool/archive/refs/tags/1.1.2.tar.gz"
+    sha256 "f1746ead7815c13be1152f0645db8ea3b277628eb0110d42a0a186db37d40a91"
 
     resource "pico-sdk" do
```
Comment (carlocab) [inline Formula/picotool.rb:10]: I think we can update this to the latest version.

Comment (Porkepix) [reply Formula/picotool.rb:10]: I thought there was some code introduced a while ago to also automatically update resources on version bumps?

Comment (Porkepix) [reply Formula/picotool.rb:10]: Also, there isn't a zip "with submodules" like for the previous one, only  the source code zips: https://github.com/raspberrypi/pico-sdk/releases/

Comment (chenrui333) [reply Formula/picotool.rb:10]: bumped to the use the latest release.

**Comment by chenrui333** [comment]
should be good for the merge now.

**Review by carlocab — APPROVED** [review]

**Pull request merged**